xarray/tests/test_variable.py

```diff
@@ -785,36 +785,69 @@ def test_getitem_error(self):
         with raises_regex(IndexError, "Dimensions of indexers mis"):
             v[:, ind]
 
-    def test_pad(self):
+    @pytest.mark.parametrize(
+        "mode",
+        [
+            "mean",
+            pytest.param("median", marks=pytest.mark.xfail),
+            pytest.param("reflect", marks=pytest.mark.xfail),
+            "edge",
+            pytest.param("linear_ramp", marks=pytest.mark.xfail),
```

Comment:
`linear_ramp` passes locally with dask 2.9.1. Is the xfail needed for an older dask version? If so, let's mention that